tests/myst.spec.ts

```diff
@@ -12,11 +12,23 @@ type TestCase = {
   description?: string;
   skip?: boolean;
   mdast: Root;
-  myst: string;
-  html: string;
+  myst?: string;
+  html?: string;
 };
 
-const directory = 'tests/myst';
+// Comment out to make these fail
+const SKIP_TESTS = [
```

Comment:
Much of this is copied from `commonmark.spec.ts` - those tests are pretty much obsolete now.